app/policies/collection_version_policy.rb

```diff
@@ -19,6 +30,10 @@ def destroy?
     (administrator? || manages_collection?(record.collection)) && record.persisted? && record.version_draft?
   end
 
+  def deposit?
+    record.updatable? && show?
+  end
```

Comment:
Only allow depositing into a collection if it's in an updatable state (decommissioned collections are not) and if the user can see it (incl. managers, reviewers, depositors).